scripts/us_bls/jolts/BLSJolts.tmcf

```diff
@@ -0,0 +1,12 @@
+Node: E:BLSJolts->E0
+typeOf: Country
+dcid: country/USA
+
+Node: E:BLSJolts->E1
+typeOf: dcs:StatVarObservation
+variableMeasured: C:BLSJolts->StatisticalVariable
+measurementQualifier: C:BLSJolts->SeasonalAdjustment
```

Comment:
This should go into the StatVar instead.

Reply:
Moved